packages/grafana-ui/src/components/BigValue/BigValue.tsx

```diff
@@ -87,23 +89,42 @@ export class BigValue extends PureComponent<Props> {
   };
 
   render() {
-    const { onClick, className, hasLinks, theme } = this.props;
+    const { onClick, className, hasLinks, theme, textMode, parentOrientation } = this.props;
     const layout = buildLayout(this.props);
     const panelStyles = layout.getPanelStyles();
     const valueAndTitleContainerStyles = layout.getValueAndTitleContainerStyles();
     const valueStyles = layout.getValueStyles();
     const titleStyles = layout.getTitleStyles();
     const textValues = layout.textValues;
+    const percentChange = this.props.value.percentChange;
+    const percentChangeNaN = Number.isNaN(percentChange);
+    const percentChangeString =
+      percentChange?.toLocaleString(undefined, { style: 'percent', maximumSignificantDigits: 3 }) ?? '';
+    const percentChangeIcon =
+      percentChange && (percentChange > 0 ? 'arrow-up' : percentChange < 0 ? 'arrow-down' : undefined);
+    const percentChangeStyles = layout.getValueStyles(true);
 
     // When there is an outer data link this tooltip will override the outer native tooltip
     const tooltip = hasLinks ? undefined : textValues.tooltip;
+    const iconDim =
+      parentOrientation === VizOrientation.Horizontal ? layout.maxTextHeight / 3 : layout.maxTextHeight / 8;
```

Comment:
I don't see how VizRepeater orientation matters here (also removing this property in https://github.com/grafana/grafana/pull/79389) 

and this check also makes does work for the "Auto" orientation (where VizRepeater will automatically decide orientation)

Reply:
I was attempting to use VizOrientation to determine an appropriate icon and font size to utilize vertical space. There is probably a better way to handle this, especially considering parentOrientation is going away. I'll look at a better approach here.